### DISCUSSION TITLE
thinkpad-t14-gen2-amd-review/

### DISCUSSION THREAD

**Discussion by giscus** [n0vad3v/blog-comments#12] [Ideas]
# thinkpad-t14-gen2-amd-review/

本文在 2023-02-06 更新了 Possible Hardware bug 章节 当我还在 PingCAP 工作的时候，我的主力电脑是从大学时候一直流传下来的 ThinkPad X1 Carbon Gen5 ，但是由于当时买的是最低配置的版本，内存只有 8G

https://nova.moe/thinkpad-t14-gen2-amd-review/

**Comment by manorjust** [comment]
"笔记本常年不会手动关机，现在遇到的问题是它会在半夜偶发自动关机（可能没有关机，各个电源灯都是亮的，但是外接显示屏显示绿色，笔记本显示屏显示纯黑色，键盘鼠标没有任何响应）的问题，遇到这种问题一般只能长按电源硬关机"

------推测是后加内存的原因，我也是这个现象，后来把后加的内存拆了就没发生过了。

**Reply by n0vad3v** [reply]
不过这里很神奇的一点是，我把两个硬盘对换了之后这个问题就少见了。
之后换回过自带的硬盘跑 Linux ，这个问题也变得少见了，目前换成了 980PRO 之后也很少遇到了，感觉是个很玄学的问题 🤣

**Reply by n0vad3v** [reply]
好像真的是内存的问题，质保了内存之后已经两个月没发生过自动关机的情况了（还好内存有 3 年质保）😂.

**Reply by manorjust** [reply]
我的这个电脑2个月前已经被玩坏了😂。自己改了4k 500nits 屏幕，长期高负载，lcd的供电烧了，彻底黑屏。
已换macbook

**Reply by n0vad3v** [reply]
啊这... 🤣
